### PR TITLE
PPI summer 2023 refresh 

### PR DESCRIPTION
**Pull request by irinazherko**

### CODE REVIEW

File: PPI/manual_work/Jul_2023/manual_tables.sql [hunk 1]
```diff
@@ -0,0 +1,383 @@
+-- concept_manual backup
```
Comment (TinyRickC137) [inline PPI/manual_work/Jul_2023/manual_tables.sql:1]: Backups are no longer needed after the introduction of common manual tables - to be removed

File: PPI/manual_work/Jul_2023/manual_tables.sql [hunk 2]
```diff
@@ -0,0 +1,383 @@
...
+                       'concept_relationship_manual_backup_' || update);
+    END
+$body$;
+
+SELECT * FROM dev_ppi.concept_relationship_manual_backup_2023_07_25;
+
+TRUNCATE concept_manual;
+TRUNCATE concept_relationship_manual;
+
+-- deprecate previous Mental Health and Well-Being Module concepts --229
```
Comment (TinyRickC137) [inline PPI/manual_work/Jul_2023/manual_tables.sql:34]: It is a prominent example of bad coding. Collect these concepts in some kind of manual table and insert them into concept_manual from there. How did you assess that every concept from the module has been deleted?

File: PPI/manual_work/Jul_2023/manual_tables.sql [hunk 3]
```diff
@@ -0,0 +1,383 @@
...
+'PPI' AS vocabulary_id,
+'Answer' AS concept_class_id,
+null AS source_standard_concept,
+answer_code AS concept_code,
+CURRENT_DATE AS valid_start_date,
+TO_DATE('20991231','yyyymmdd') AS valid_end_date,
+NULL AS invalid_reason
+from ehh_pr;
+
+--insert questions-answers relationships into concept_relationship_manual
```
Comment (TinyRickC137) [inline PPI/manual_work/Jul_2023/manual_tables.sql:212]: Why is this part commented?

File: PPI/manual_work/Jul_2023/process_source_table.sql [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+update ehh_pr set question_code = trim(question_code) ;
+update ehh_pr set answer_name = trim(answer_name) ;
+update ehh_pr set question_name = trim(question_name) ;
+
+SELECT * FROM ehh_pr; --408
+
+SELECT DISTINCT question_code, question_name FROM ehh_pr where length (question_name) > 255;
+SELECT DISTINCT answer_code, answer_name FROM ehh_pr where length (answer_name) > 255;
+
+--update for already existing codes
```
Comment (TinyRickC137) [inline PPI/manual_work/Jul_2023/process_source_table.sql:139]: UPDATE ehh_pr SET question_code = concat(question_code, '-', 1)
WHERE question_code IN (...);

File: PPI/manual_work/Jul_2023/manual_tables.sql [hunk 5]
```diff
@@ -0,0 +1,383 @@
...
+       c.vocabulary_id,
+       c1.vocabulary_id,
+       cr.relationship_id,
+       cr.valid_start_date,
+       current_date as valid_end_date,
+       'D' as invalid_reason
+FROM concept c
+LEFT JOIN concept_relationship cr on c.concept_id = cr.concept_id_1
+LEFT JOIN concept c1 on cr.concept_id_2 = c1.concept_id
+WHERE c.concept_code in (SELECT concept_code FROM concept_manual)
```
Comment (TinyRickC137) [inline PPI/manual_work/Jul_2023/manual_tables.sql:122]: Never ever use concept code alone for the identification of concepts. Currently, all the manual tables are recreated by fastrecreate, therefore, this code is very dangerous

File: PPI/manual_work/Jul_2023/manual_tables.sql [hunk 6]
```diff
@@ -0,0 +1,383 @@
...
+    END
+$body$;
+
+SELECT * FROM dev_ppi.concept_relationship_manual_backup_2023_07_25;
+
+TRUNCATE concept_manual;
+TRUNCATE concept_relationship_manual;
+
+-- deprecate previous Mental Health and Well-Being Module concepts --229
+INSERT INTO dev_ppi.concept_manual (
```
Comment (TinyRickC137) [inline PPI/manual_work/Jul_2023/manual_tables.sql:35]: Why referencing dev_ppi?

File: PPI/manual_work/Jul_2023/manual_tables.sql (hunk 1 above)
Comment (irinazherko) [reply PPI/manual_work/Jul_2023/manual_tables.sql]: done

File: PPI/manual_work/Jul_2023/manual_tables.sql (hunk 2 above)
Comment (irinazherko) [reply PPI/manual_work/Jul_2023/manual_tables.sql]: done

File: PPI/manual_work/Jul_2023/manual_tables.sql (hunk 3 above)
Comment (irinazherko) [reply PPI/manual_work/Jul_2023/manual_tables.sql]: it was done for reverse relationships.
now resolved.

File: PPI/manual_work/Jul_2023/process_source_table.sql (hunk 4 above)
Comment (irinazherko) [reply PPI/manual_work/Jul_2023/process_source_table.sql:139]: done

**Review by TinyRickC137 — APPROVED** [review]

**Review by TinyRickC137 — APPROVED** [review]

**Review by hardhouse — APPROVED** [review]

**Pull request merged**